apps/public-docsite-v9/src/Concepts/SSR/NextJSAppDir.stories.mdx

```diff
@@ -0,0 +1,108 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Server-Side Rendering/Next.js appDir setup" />
+
+## Next.js appDir router
+
+For basic instructions on getting Next.js set up, see [Getting Started](https://nextjs.org/docs/getting-started).
+
+1. Get a basic next.js setup running, rendering a page from the `app` folder, as guided by the tutorial.
+2. Add the Fluent UI dependencies: `@fluentui/react-components`.
+3. Add the SWC plugin to add the client directive for our library: `fluentui-next-appdir-directive`. See
+   [the repo](https://github.com/sopranopillow/fluentui-nextjs-appdir-plugin) for more information
```

Comment:
we should bring this plugin to fluent-contrib monorepo as our official tooling support for nextjs

Reply:
I discussed this with Shift, since the plugin is written in Rust and there's no handling for such packages we went ahead and kept it under my handle. What are your thoughts on this?

Reply:
> there's no handling for such packages

- fluent contrib uses nx, nx is language/tech agnostic  - we will need to add some additional setup. 
- for future feel free to bring these kind of topics to v-build 🙌

Reply:
feel free to resolve this convo to be able to merge